### PR TITLE
docs(python): Document sys.exit integration

### LINKED ISSUE #11043: Document Python's `SysExitIntegration`
### Core or SDK?

Platform/SDK

### Which part? Which one?

Python

### Description

Documentation for the `SysExitIntegration` added in https://github.com/getsentry/sentry-python/pull/3401

### Suggested Solution

Write the docs!

### PR DESCRIPTION
**Pull request by szokeasaurusrex**
**Converted to draft pending release of the [integration](https://github.com/getsentry/sentry-python/pull/3401).** Add the version number that the integration was released in prior to merge.

## DESCRIBE YOUR PR
Add documentation for the sys.exit integration, introduced in https://github.com/getsentry/sentry-python/pull/3401.

Closes #11043

## IS YOUR CHANGE URGENT?  

None: Not urgent, can wait up to 1 week+

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

**Review by sl0thentr0py — APPROVED** [review]

File: docs/platforms/python/integrations/sys_exit/index.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: sys.exit
+description: Learn how to use Sentry to capture sys.exit calls
```
Comment (vivianyentran) [inline docs/platforms/python/integrations/sys_exit/index.mdx:3]:
```suggestion
description: Learn how to use Sentry to capture sys.exit calls.
```

File: docs/platforms/python/integrations/sys_exit/index.mdx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## Install
+
+```bash
+pip install --upgrade "sentry-sdk"
+```
+
+## Configure
+
+The `SysExitIntegration` is disabled by default, and the SDK only captures `SystemExit` exceptions automatically if this integration is manually enabled.
+
```
Comment (vivianyentran) [inline docs/platforms/python/integrations/sys_exit/index.mdx:17]:
```suggestion

A `sys.exit` call is considered "successful" when the function is passed a value of `0` or `None`. Passing any other value results in an "unsuccessful" exit.

```

File: docs/platforms/python/integrations/sys_exit/index.mdx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import sentry_sdk
+from sentry_sdk.integrations.sys_exit import SysExitIntegration
+
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+    integrations=[SysExitIntegration(capture_successful_exits=True)],
+)
+```
+
+A `sys.exit` call is considered "successful" when the function is passed a value of `0` or `None`. Passing any other value results in an "unsuccessful" exit.
```
Comment (vivianyentran) [inline docs/platforms/python/integrations/sys_exit/index.mdx:44]:
```suggestion
```
I'd move this up to before the code samples

**Review by vivianyentran — APPROVED** [review]
Looks good, thanks for adding this!

**Comment by szokeasaurusrex** [comment]
thanks for the suggestions @vivianyentran!

**Pull request merged**